android/brave_java_sources.gni

```diff
@@ -51,7 +51,7 @@ brave_java_sources = [
   "../../brave/android/java/org/chromium/chrome/browser/ntp_sponsored_images/SponsoredTab.java",
   "../../brave/android/java/org/chromium/chrome/browser/ntp_sponsored_images/NewTabPageListener.java",
   "../../brave/android/java/org/chromium/chrome/browser/ntp_sponsored_images/RewardsBottomSheetDialogFragment.java",
-  "../../brave/android/java/org/chromium/chrome/browser/ntp_sponsored_images/NTPSponsoredImagesBridge.java",
+  "../../brave/android/java/org/chromium/chrome/browser/ntp_sponsored_images/NTPBackgroundImagesBridge.java",
```

Comment:
are we going to change the java path in another commit? It should be the same for anything in ntp_sponsored_images